<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRoomConnect</class>
 <widget class="QDialog" name="ChatRoomConnect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatRoomConnect</string>
  </property>
  <widget class="QLineEdit" name="m_chatRoomName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Chat room name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_connect">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>264</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
